ambry-account/src/main/java/com/github/ambry/account/HelixAccountService.java

```diff
@@ -65,12 +56,30 @@
  *   will not update its local cache.
  * </p>
  * <p>
- *   The full set of the {@link Account} metadata are stored in a single {@link ZNRecord} or a single blob in the new way, as a
- *   simple map from a string account id to the {@link Account} content in json as a string.
+ *   Every time a mutation of account metadata is performed, a new backup file will be created for locally, in a directory
+ *   specified by the {@link HelixAccountServiceConfig#backupDir}. A backup file is created when the helix listener on
+ *   {@link #ACCOUNT_METADATA_CHANGE_TOPIC} is notified. And backup's filename could contain version and mtime information
```

Comment:
May I know what's mtime? modified time?

Reply:
Yep, I believe mtime is the last modified time for a node in zookeeper.

Reply:
Change mtime to last modified time, and hopefully this would make it more obvious.